pkg/cloud/provider.go

```diff
@@ -513,6 +521,9 @@ func getClusterProperties(node *v1.Node) clusterProperties {
 		cp.provider = kubecost.AzureProvider
 		cp.configFileName = "azure.json"
 		cp.accountID = parseAzureSubscriptionID(providerID)
+	} else if strings.HasPrefix(providerID, "scaleway") {
```

Comment:
Could you share a little more detail here? Maybe a quick comment about what a scaleway providerID label looks like?